cli/smartcontract/smart_contract.go

```diff
@@ -681,7 +681,7 @@ func invokeWithArgs(ctx *cli.Context, acc *wallet.Account, wall *wallet.Wallet,
 			}
 		}
 		if !ctx.Bool("force") {
-			return sender, cli.NewExitError(errText+". Use --force flag to "+action+" the transaction anyway.", 1)
+			return sender, cli.NewExitError(errText+".\nUse --force flag to "+action+" the transaction anyway.", 1)
 		}
 		fmt.Fprintln(ctx.App.Writer, errText+". "+process+" transaction...")
```

Comment:
Let's add newline after `.` also.

Reply:
Do you mean the last dot? Newline will be printed on exit anyway.

Reply:
No, I mean dot in the middle of the `fmt.Fprintln(ctx.App.Writer, errText+". "+process+" transaction...")` line, the case when there's a `--force` flag and we're sending the transaction anyway.

Reply:
Just for the consistency between L684 and L686.

Reply:
Uh, right, done!